src/components/azure-pg/drop-autodevops-dbs.job.ts

```diff
@@ -4,8 +4,9 @@ interface DropAutodevopsDbsJobArgs {
   secretRefName?: string;
 }
 
-// renovate: datasource=docker depName=registry.gitlab.factory.social.gouv.fr/socialgouv/docker/azure-db versioning=5.0.1
-const SOCIALGOUV_DOCKER_AZURE_DB_VERSION = "5.0.1";
+// renovate: datasource=docker depName=ghcr.io/socialgouv/docker/azure-db versioning=6.0.1
+const SOCIALGOUV_DOCKER_IMAGE = "ghcr.io/socialgouv/docker/azure-db";
+const SOCIALGOUV_DOCKER_VERSION = "6.0.1";
```

Comment:
The renovate commit should be next to the version variable

```suggestion
const SOCIALGOUV_DOCKER_IMAGE = "ghcr.io/socialgouv/docker/azure-db";
// renovate: datasource=docker depName=ghcr.io/socialgouv/docker/azure-db versioning=6.0.1
const SOCIALGOUV_DOCKER_VERSION = "6.0.1";
```